satpy/readers/seviri_l2_bufr.py

```diff
@@ -19,13 +19,14 @@
 
 
 import logging
-from datetime import timedelta
+from datetime import timedelta, datetime
 import numpy as np
 import xarray as xr
 import dask.array as da
-
+from fnmatch import fnmatch
 from satpy.readers.seviri_base import mpef_product_header
 from satpy.readers.eum_base import recarray2dict
+import os
```

Comment:
module not needed, see comment on line 61

Reply:
removed